rust/fastsim-core/src/cycle.rs

```diff
@@ -726,6 +726,19 @@ impl SerdeAPI for RustCycle {
             ),
         }
     }
+
+    /// takes an object from a url and saves it in the fastsim data directory in a rust_objects folder
+    /// WARNING: if there is a file already in the data subdirectory with the same name, it will be replaced by the new file
```

Comment:
This warning is good, and I prefer the behavior you have now and this warning being in the doc string as opposed to adding another function args.